README.md

```diff
@@ -1,114 +1,142 @@
+## Step 3: Define database and schema variables
+By default, this package runs using your destination and the `linkedin_ads` schema. If this is not where your Linkedin Ad Analytics data is (for example, if your Linkedin schema is named `linkedin_ads_fivetran`), add the following configuration to your root `dbt_project.yml` file:
 
 ```yml
 # dbt_project.yml
-
-...
-config-version: 2
-
 vars:
-    linkedin_database: your_database_name
-    linkedin_schema: your_schema_name
-
+    linkedin_ads_schema: your_schema_name
+    linkedin_ads_database: your_destination_name 
 ```
-For additional configurations for the source models, please visit the [LinkedIn Ad Analytics source package](https://github.com/fivetran/dbt_linkedin_source).
+
+## (Optional) Step 4: Additional configurations
 
 ### Switching to Local Currency
 Additionally, the package allows you to select whether you want to add in costs in USD or the local currency of the ad. By default, the package uses USD. If you would like to have costs in the local currency, add the following variable to your `dbt_project.yml` file:
 
 ```yml
 # dbt_project.yml
-
-...
-config-version: 2
-
 vars:
-    linkedin__use_local_currency: True
+    linkedin_ads__use_local_currency: True # false by default -- uses USD
 ```
 
 ### Passing Through Additional Metrics
-By default, this package will select `clicks`, `impressions`, and `costs` from the source `ad_analytics_by_creative` table to store into the `linkedin__ad_adapter` model. If you would like to pass through additional metrics to the ad adapter model, add the following configuration to your `dbt_project.yml` file:
+By default, this package will select `clicks`, `impressions`, and `cost` from `ad_analytics_by_creative` and `ad_analytics_by_campaign`. If you would like to pass through additional metrics to the end report models, add the variable configuration in the code block below to your `dbt_project.yml` file. These variables allow for the pass-through fields to be aliased (`alias`) and casted (`transform_sql`) if desired, but not required. Datatype casting is configured via a sql snippet within the `transform_sql` key. You may add the desired sql while omitting the `as field_name` at the end and your custom pass-though fields will be casted accordingly. Use the below format for declaring the respective pass-through variables:
 
 ```yml
 # dbt_project.yml
-
-...
 vars:
-    linkedin__passthrough_metrics: ['the', 'list', 'of', 'metric', 'columns', 'to', 'include'] # from LINKEDIN_ADS.AD_ANALYTICS_BY_CREATIVE
+    linkedin_ads__passthrough_metrics: 
+        - name: "new_custom_field"
+          alias: "custom_field"
+        - name: "unique_int_field"
+          alias: "field_id"
+          transform_sql: "cast(field_id as int)"
+        - name: "that_field"
```

Comment:
Based on the outcome of the question posed in the source package, we may or may not need to update this.

Reply:
updated